src/ontology/edits/oeo-physical.omn

```diff
@@ -7743,23 +7743,25 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/810"
 Class: OEO_00230021
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A photon is a portion of matter that is a light particle."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A photon is a material entity that is a light particle."@en,
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/661
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/663
 pull request: https://github.com/OpenEnergyPlatform/ontology/issues/674
 add origin
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/515
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/684
 remove origin 
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/861"@en,
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/861
+classify as material entity
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1165"@en,
```

Comment:
Please include also the issue in the term tracker

Reply:
I never include the issue in the term tracker. I don't do that on purpose because the issue number is retraceable via the PR and I thinks it'S good to keep it as short as possible.

Reply:
Somehow I've never noticed that. Maybe I haven't looked closely enough at your term trackers before. ;)

Should we adapt https://github.com/OpenEnergyPlatform/ontology/wiki/term-tracker-item then?